plugins/modules/digital_ocean_floating_ip.py

```diff
@@ -270,6 +270,17 @@ def create_floating_ips(module, rest):
     if module.params['droplet_id'] is not None:
         payload["droplet_id"] = module.params['droplet_id']
 
+    # Get existing floating IPs
+    response = rest.get('floating_ips/')
+    status_code = response.status_code
+    json_data = response.json
+
+    # Exit unchanged if any of them are assigned to this Droplet already
+    if status_code == 200:
+        for floating_ip in json_data['floating_ips']:
```

Comment:
Consider using `.get()` in this section of code, and checking to be sure the response JSON is not `None`.